### PR TITLE
[descheduler] Documentation fixes

### PR DESCRIPTION
**Pull request by unreturned**
## Description
Descriptions of strategies in the Russian version are mixed up in places.
Style fixes.

## Why do we need it, and what problem does it solve?
Correct documentation is the key to success.

## Checklist
- [ ] The code is covered by unit tests.
- [ ] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [ ] Changes were tested in the Kubernetes cluster manually.

## Changelog entries
```changes
section: descheduler
type: chore
summary: Fix documentation.
impact_level: low
```

### CODE REVIEW

**Comment by z9r5** [comment]
@unreturned Thank for the contribution.

**Comment by z9r5** [comment]
@unreturned Fix DCO test please ([details...](https://github.com/deckhouse/deckhouse/pull/6640/checks?check_run_id=18970714174))
TLDR: `git rebase HEAD~1 --signoff`

**Comment by unreturned** [comment]
@z9r5 Fixed

**Review by z9r5 — APPROVED** [review]

**Comment by z9r5** [comment]
Cherry-picked into release-1.54, release-1.55
